cli/spinner.ts

```diff
@@ -4,9 +4,26 @@ const encoder = new TextEncoder();
 
 const LINE_CLEAR = encoder.encode("\r\u001b[K"); // From cli/prompt_secret.ts
 const COLOR_RESET = "\u001b[0m";
-const DEFAULT_SPEED = 75;
+const DEFAULT_INTERVAL = 75;
 const DEFAULT_SPINNER = ["⠋", "⠙", "⠹", "⠸", "⠼", "⠴", "⠦", "⠧", "⠇", "⠏"];
 
+// This is a hack to allow us to use the same type for both the color name and an ANSI escape code.
+// deno-lint-ignore ban-types
+type Ansi = string & { };
+type Color = 'black' | 'red' | 'green' | 'yellow' | 'blue' | 'magenta' | 'cyan' | 'white' | 'gray' | Ansi;
```

Comment:
@iuioiua I used this typescript hack to allow for type suggestions on colors and allowing for a ANSI string, is this ok?

See https://github.com/microsoft/TypeScript/issues/29729#issuecomment-460346421